examples/java-matter-controller/java/src/com/matter/controller/commands/discover/DiscoverCommissionersCommand.kt

```diff
@@ -1,35 +1,30 @@
-/*
- *   Copyright (c) 2022 Project CHIP Authors
- *   All rights reserved.
- *
- *   Licensed under the Apache License, Version 2.0 (the "License");
- *   you may not use this file except in compliance with the License.
- *   You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- *   Unless required by applicable law or agreed to in writing, software
- *   distributed under the License is distributed on an "AS IS" BASIS,
- *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *   See the License for the specific language governing permissions and
- *   limitations under the License.
- *
- */
-
-package com.matter.controller.commands.discover;
-
-import chip.devicecontroller.ChipDeviceController;
-import com.matter.controller.commands.common.CredentialsIssuer;
-import com.matter.controller.commands.common.MatterCommand;
-
-public final class DiscoverCommissionersCommand extends MatterCommand {
-  public DiscoverCommissionersCommand(
-      ChipDeviceController controller, CredentialsIssuer credsIssuer) {
-    super(controller, "commissioners", credsIssuer);
-  }
-
-  @Override
-  protected final void runCommand() {
-    // mCommissionableNodeController.DiscoverCommissioners();
-  }
-}
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.discover
+
+import chip.devicecontroller.ChipDeviceController
+import com.matter.controller.commands.common.CredentialsIssuer
+import com.matter.controller.commands.common.MatterCommand
+
+class DiscoverCommissionersCommand(
+  controller: ChipDeviceController, credsIssuer: CredentialsIssuer?
+) : MatterCommand(controller, "commissioners", credsIssuer) {
+  override fun runCommand() {
+    // mCommissionableNodeController.DiscoverCommissioners();
```

Comment:
A comment as to why this was commented out would help. I also realize this is existing code ... do we know history here?

Reply:
This command is not implemented, remove the comment